flake.nix

```diff
@@ -98,6 +99,17 @@
           '';
         };
 
+        process-compose-flake = {
+          baseUrl = "https://github.com/Platonic-Systems/process-compose-flake/blob/main";
+          intro = ''
+            Declare one or more process-compose configurations using options.
+
+            Generates a configuration for [process-compose](https://github.com/F1bonacc1/process-compose) in `packages` and `apps` for `nix run`.
```

Comment:
It wraps `process-compose` with the config and creates a package and apps output. Not sure if the wording reflects this.